docker/dockerconfig.json

```diff
@@ -0,0 +1,49 @@
+{
+  "DiskCache": {
+    "cachePath": "./docker-state/DiskCacheData",
+    "localSymbolDirs": [
+    ],
+    "maxSizeMB": 200,
+    "port": 8888,
+    "symbolURLs": [
+      "https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-public/v1/"
+    ],
+    "log": {
+      "path": "./docker-state/diskcache.log",
```

Comment:
I put the state in the current directory which is mounted. Another way to do this is create a data volume and put the stage there.

"State" here includes log files and cache data.